### DISCUSSION TITLE
Incognito

### DISCUSSION THREAD

**Discussion by Terrinn** [3kh0/ext-remover#160] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is it possible to open an incognito window on a school chromebook

**Comment by ghost** [comment]
no

**Comment by 3kh0** [comment, marked as answer]
No, there is a bug where you can try. Go to the address bar and click on this icon

![image](https://user-images.githubusercontent.com/58097612/198674765-1fd11253-0bc0-472b-8d06-0631156cd327.png)

It might not have the blue dot, click on add current tab, and right click on the new entry and click on Open In incognito.

**Reply by ghost** [reply]
uh oh! turns out this will mess your chromebook up ( I tried it ) don't do it!! It will reset your chromebook and make everything slow and blank
